pkg/service/tools/tools.go

```diff
@@ -274,3 +277,109 @@ func (t *Tools) CreateConfig(_ context.Context, filePath string, configData stri
 	t.logger.Info("Config file generated successfully")
 	return nil
 }
+
+// Normalise initiates the normalise process for normalising the test cases.
```

Comment:
This will significantly change after resolving above comments so will review this file once above comments are addressed .